internal/pkg/service/stream/storage/level/local/encoding/pipeline_test.go

```diff
@@ -33,14 +33,13 @@ func TestEncodingPipeline_Basic(t *testing.T) {
 	ctx := context.Background()
 	logger := log.NewDebugLogger()
 	clk := clock.New()
-	slice := test.NewSlice()
 
-	cfg := encoding.NewConfig()
-	cfg.Encoder.Factory = encoder.FactoryFn(dummyEncoderFactory)
+	slice := test.NewSlice()
+	slice.LocalStorage.Encoding.Encoder.Factory = encoder.FactoryFn(dummyEncoderFactory)
```

Comment:
How can storage contain Encoding? Storage stores bytes. Cannot they be mixed? (Encoded, not encoded) within `LocalStorage`.